### PR TITLE
readonly: add page

### PR DESCRIPTION
**Pull request by parth93QA**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/readonly.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# readonly
+
+> In Linux, the readonly command is used to create or modify read-only variables within a shell script.
+> When a variable is marked as read-only using readonly, its value cannot be changed or modified by subsequent commands within the script.
+> This can be useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
+> More information: <https://man7.org/linux/man-pages/man1/readonly.1p.html>.
```
Comment (reinhart1010) [inline pages/common/readonly.md]:
```suggestion
> More information: <https://manned.org/readonly.1p>.
```
We currently use manned.org for manpage links. See #5143.

**Comment by reinhart1010** [comment]
Hi, it seems that `readonly` has become [a standard POSIX command](https://manned.org/readonly.1p) which means that it's available for BSD and macOS, too.

But the POSIX version doesn't support the `-u` flag, so I recommend creating `linux/readonly.md` too (with the `-u` documentation) and remove the `-u` documentation

Note that we usually use manned.org for manpage-related links, see #5143. But I haven't found the appropriate manned.org link for the Linux version of the `readonly` command.

**Comment by parth93QA** [comment]
Hi @reinhart1010 I have made the changes with the -p flag with the appropriate link from manned.org for the POSIX version.

File: pages/common/readonly.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- To create a read-only variable:
+
+`readonly {{variable_name}}={{value}}`
+
+- To mark a variable as read-only:
+
+`readonly {{existing_variable}}`
+
+- Write to the standard output the names and values of all read-only variables:
```
Comment (reinhart1010) [inline pages/common/readonly.md:16]:
```suggestion
- [p]rint to the standard output the names and values of all read-only variables:
```
Maybe this work better?

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
I think just one more change to request.

File: pages/common/readonly.md (hunk 1 above)
Comment (parth93QA) [reply pages/common/readonly.md]: pushed the changes @reinhart1010

**Comment by parth93QA** [comment]
Hi @reinhart1010 I have commited the requested changes.

File: pages/common/readonly.md (hunk 1 above)
Comment (parth93QA) [reply pages/common/readonly.md]: This is also done @reinhart1010

File: pages/common/readonly.md (hunk 2 above)
Comment (parth93QA) [reply pages/common/readonly.md]: Yes thanks for the suggestion @reinhart1010

**Review by reinhart1010 — APPROVED** [review]
Thanks for your work! However, we need to wait for another maintainer to approve this before finally merged into this repo.

File: pages/common/readonly.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# readonly
+
+> In Linux, the readonly command is used to create or modify read-only variables within a shell script.
+> When a variable is marked as read-only using readonly, its value cannot be changed or modified by subsequent commands within the script.
+> This can be useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
```
Comment (blueskyson) [inline pages/common/readonly.md:5]:
```suggestion
> Create or modify read-only variables within a shell script, preventing the variable changed by subsequent commands.
> This is useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
```

File: pages/common/readonly.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# readonly
+
+> Create or modify read-only variables within a shell script, preventing the variable changed by subsequent commands.
+> This is useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
```
Comment (blueskyson) [inline pages/common/readonly.md:4]:
```suggestion
> Create or modify read-only variables within a shell script, preventing the variable from being changed by subsequent commands.
> This is useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
```

File: pages/common/readonly.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# readonly
+
+> Create or modify read-only variables within a shell script, preventing the variable from being changed by subsequent commands.
+> This is useful when you want to ensure that a variable retains a constant value throughout the execution of a script.
+> More information: <https://manned.org/readonly.1p>.
+
+- To create a read-only variable:
```
Comment (blueskyson) [inline pages/common/readonly.md:7]:
```suggestion
- Create a read-only variable:
```

File: pages/common/readonly.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+`readonly {{variable_name}}={{value}}`
+
+- To mark a variable as read-only:
```
Comment (blueskyson) [inline pages/common/readonly.md:11]:
```suggestion
- Mark a variable as read-only:
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**